app/livechat/server/business-hour/Single.ts

```diff
@@ -0,0 +1,26 @@
+import moment from 'moment';
+
+import { ILivechatBusinessHour, LivechatBussinessHourTypes } from '../../../../definition/ILivechatBusinessHour';
+import { AbstractBusinessHour, IBusinessHour } from './AbstractBusinessHour';
+
+export class SingleBusinessHour extends AbstractBusinessHour implements IBusinessHour {
+	async saveBusinessHour(businessHourData: ILivechatBusinessHour): Promise<void> {
+		businessHourData.timezone = {
+			name: '',
+			utc: moment().utcOffset() / 60,
+		};
+		if (!businessHourData._id) {
```

Comment:
This check should be the first step at the beginning of the method since you may return.